### PR TITLE
never-download got readded

### LINKED ISSUE #43338: virtualenv never-download
https://github.com/saltstack/salt/blob/0e262b8c0d9f7ee4932bcdc301f5e4a91a5ae5ee/salt/modules/virtualenv_mod.py#L202

I have virtualenv 15.1.0 and this option is not deprecated

`  --no-download, --never-download
                        Do not download preinstalled packages from PyPI.`

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?
https://github.com/pypa/virtualenv/pull/840

### What issues does this PR fix or reference?
Fixes #43338 

### Tests written?

No

### CODE REVIEW

**Review by rallytime — APPROVED** [review]

**Comment by rallytime** [comment]
Ah, @gtmanfred this is failing a unit test. Can you fix that up? :)

**Comment by gtmanfred** [comment]
Done 👍

File: tests/unit/modules/virtualenv_test.py [hunk 1]
```diff
@@ -112,11 +112,9 @@ def test_issue_6030_deprecated_never_download(self):
 
                 # Are we logging the deprecation information?
                 self.assertIn(
-                    'INFO:The virtualenv \'--never-download\' option has been '
-                    'deprecated in virtualenv(>=1.10), as such, the '
-                    '\'never_download\' option to `virtualenv.create()` has '
-                    'also been deprecated and it\'s not necessary anymore.',
-                    handler.messages
```
Comment (rallytime) [inline tests/unit/modules/virtualenv_test.py]: This `handler.messages` line is still needed for the comparison.

**Comment by gtmanfred** [comment]
woops, sorry forgot about this.

Thanks!
Daniel

**Pull request merged**